src/app/containers/Navigation/index.stories.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import NavigationContainer from '.';
+import services from '../../lib/config/services';
+import { ServiceContextProvider } from '../../contexts/ServiceContext';
+
+const stories = storiesOf('Navigation', module);
+
+Object.keys(services)
+  .filter(service => ['default', 'news', 'persian'].indexOf(service) === -1)
```

Comment:
So we dont have to maintain this list, we can make it dependant on the config :)
```suggestion
  .filter(service => services[service].navigation)
```